site/components/Playground.tsx

```diff
@@ -53,23 +62,102 @@ export function Playground() {
       result: output.result(),
       result_hash: output.result_hash(),
       hashes: output.hashes(),
-      // selfDestructed: output.self_destructed(),
+      selfDestructed: output.self_destructed(),
       readAuth: output.read_auth(),
     }, null, 2))
   })
 
-  const verify = useAsyncCallback(() => {
-    if (!output) {
+  // Uses the /prove endpoint
+  const prove_server = useAsyncCallback(async () => {
```

Comment:
`useAsyncCallback` automatically captures and displays errors to the user (that's part of its purpose). So no need to catch manually.